### PR TITLE
[WIP] Add temp import_tables for FCA Import

### PR DESCRIPTION
**Pull request by JiggyPete**
This PR introduces three tables to facilitate the migration of the fca import script to being something that can be run from within the admin area of RAD. 

This will allow Sandrine to run the import rather than take up dev time.

### CODE REVIEW

File: app/models/lookup/import/adviser.rb [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+      def self.table_name
+        "lookup_import_#{super}"
+      end
+
+      def self.fca_import_copy_statement
+        "COPY #{table_name} (reference_number, name, created_at, updated_at) FROM stdin;"
+      end
+
+      def self.truncate_sql
+        "TRUNCATE #{table_name};"
```
Comment (munckymagik) [inline app/models/lookup/import/adviser.rb]: We can access a Rails provided way of truncating tables:

https://github.com/rails/rails/blob/v4.2.5.2/activerecord/lib/active_record/connection_adapters/postgresql_adapter.rb#L261

```ruby
ActiveRecord::Base.connection.truncate(table_name)
```

Comment (JiggyPete) [inline app/models/lookup/import/adviser.rb]: We can, however that will put the truncate in a different transaction to the copy.

**Comment by oneKelvinSmith** [comment]
:thumbsup:

**Comment by paulanthonywilson** [comment]
18 months old and superseded.

**Pull request closed without merging**